src/actix/certificate_helpers.rs

```diff
@@ -148,13 +151,14 @@ pub fn actix_tls_server_config(settings: &Settings) -> Result<ServerConfig> {
     // Verify client CA or not
     let config = if settings.service.verify_https_client_certificate {
         let mut root_cert_store = RootCertStore::empty();
-        let ca_certs: Vec<Vec<u8>> = with_buf_read(&tls_config.ca_cert, |rd| {
-            rustls_pemfile::certs(rd)
-                .map_ok(|cert| cert.to_vec())
-                .collect()
+        let ca_certs: Vec<CertificateDer> = with_buf_read(&tls_config.ca_cert, |rd| {
+            rustls_pemfile::certs(rd).collect()
         })?;
-        root_cert_store.add_parsable_certificates(&ca_certs[..]);
-        config.with_client_cert_verifier(AllowAnyAuthenticatedClient::new(root_cert_store).boxed())
```

Comment:
the guide does not help with the removal of `AllowAnyAuthenticatedClient`

so I followed this PR https://github.com/rustls/rustls/pull/1368